docs/config-file/v2.rst

```diff
@@ -64,7 +64,7 @@ This is to avoid typos and provide feedback on invalid configurations.
 
 .. contents::
    :local:
-   :depth: 1
+   :depth: 3
```

Comment:
I wonder if this will spam the TOC too much -- seems fine tho?

Reply:
I was between 2 or 3, but wanted to have `build.apt_images` and `sphing.builder` visible instead of just having `sphinx` and `build`, but no strong opinion.